src/main/java/core/basesyntax/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService {
+
+    @Override
+    public List<String> readFromFile(String fromFilePath) {
+        try {
+            return Files.readAllLines(Path.of(fromFilePath));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from file "
```

Comment:
Consider using a more specific exception instead of RuntimeException, such as FileNotFoundException or a custom exception.